nldcsc/flask_plugins/flask_redis.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, app=None, redis_url: str = None, **kwargs):
         self.redis_cache_db = int(os.getenv("REDIS_CACHE_DB", 0))
         self.redis_kwargs = getenv_dict("REDIS_KWARGS", None)
 
-        if app is not None:
+        if app is not None or ignore_app_init:
```

Comment:
This would fail if app is None, right?

Reply:
No, since it passes app=None to .init_app() which runs normally until `app.redis_client = self` which is skipped. Essentially it would return a normal instance of the FlaskRedis class without tying it to "app". I'll add a check to ensure ignore_app_init isn't used with app as an extra precaution. 

https://github.com/NLDCSC/racoon/blob/197f03f0b308e33cb17b04ff7940e396fa80bf64/racoon/celery_app/racoon_daemon.py#L84

Reply:
Cool

Reply:
Made it a bit clearer (I hope). But another example: https://github.com/NLDCSC/racoon/blob/197f03f0b308e33cb17b04ff7940e396fa80bf64/racoon/celery_app/racoon_daemon.py#L93C1-L101C2 could now be replaced with: 
```
from nldcsc...import FlaskRedis
redis_client = FlaskRedis(init_standalone=True)
``` 
Most of the time, Celery and Flask will use the same configuration for plugins (like redis, kibana, elastic, etc) so it is nice to reuse that when possible.